libyul/backends/evm/EVMDialect.cpp

```diff
@@ -302,6 +317,9 @@ BuiltinFunctionForEVM const* EVMDialect::builtin(YulString _name) const
 
 bool EVMDialect::reservedIdentifier(YulString _name) const
 {
+	if (m_objectAccess)
```

Comment:
Had to re-read the actual source code (not this PR). It seems for the other builtins, we have reserved them in Yul entirely, not just for the object version.

If we do it here, are we afraid some people use `verbatim*` in inline assembly? We should aim to reserve it in 0.9.0 at least, if so. (Or perhaps by that time verbatim will be supported in inline assembly?)

Reply:
Sounds good!

Reply:
I'm confused, what is the verdict? We lave it as-is and change it in 0.9.0?

Reply:
Yes!